src/app/views/scorecards/charts/shift.js

```diff
@@ -140,30 +152,36 @@ define([
       let width = baseWidth - margin.left - margin.right;
       let height = baseHeight - margin.top - margin.bottom;
 
-      const svg = rootElm.append('svg')
+      const svg = rootElm
+        .append('svg')
         .attr('viewBox', `0 0 ${baseWidth} ${baseHeight}`)
         .append('g')
-          .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+        .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
-      const x = d3.scale.linear()
-        .range([0, width])
-        .domain(years);
+      const x = d3.scaleLinear().range([0, width]).domain(years);
```

Comment:
Same as above comments: Only minor changes to this file to update d3 V3 syntax to d3 V4 since it's better supported/documented.